script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -29,74 +29,75 @@ Perhaps it's just me, but her bow doesn't seem as sharp as usual.
 <Page9>
 "...I wonder if she's worried about me. However, this kind of thing really does happen all the time."
 <Page10>
-I appreciate the concern, but it's nothing to be fussed about.
-If there is anything to be concerned about,
+I appreciate the concern, but it's nothing to be worried about.
+If there's anything to be concerned about,
 it would be a serious wound like that, no matter how you look at it.
 <Page11>
 "...But using duct tape really doesn't sit right with me. It makes my blood run cold just by looking at it. The next time I see her, I'll―――"
 <Page12>
-I stop just before I was able to finish my sentence.
+I stop just before I'm able to finish my sentence.
 There is no 'next time'.
 We'll never see each other again.
 There's no way I would get involved with such a dangerous woman.
 <Page13>
-"...That's right. I decided that would be for the best."
+"...That's right. It's obviously for the best."//「……そうだ。それがいいに決まってる」
 I swallow the undeniable fact.
 Today is Monday.
-The beginning of the week. A good chance to start fresh.
+The beginning of the week. A good chance for a fresh start.
 <Page14>
 I wash my face and fix my clothes before entering the living room.
 Akiha is sitting on the sofa.
-It is just past 7 AM, but it seems that she already finished breakfast and is now gracefully enjoying a cup of tea.
+It's just past 7 AM, but it seems like she has already finished breakfast and is now gracefully enjoying a cup of tea.
 <Page15>
 "Oh, good morning, Nii-san. You are up early this morning."
-Whether it was because she was pleased to see me up so early, Akiha seems to be in a good mood.
+Possibly because she was pleased to see me up so early, Akiha seems to be in a good mood.
 "Good morning. Likewise, you seem to be up early as alwa―――"
 <Page16>
-...And then. #I suddenly feel lightheaded and push down on the corner of my eyes.
-And even then, the dizziness won't go away.
-Without knowing what's going on, my body temperature rises and―――
+...And then. 
```

Comment:
Unfortunately these # characters are a necessary construct. Although I'd probably agree with splitting these, this is an engine limitation. Please revert this

Reply:
Reverted